src/common/tests/abt_perf.c

```diff
@@ -1,38 +1,36 @@
 /**
- * (C) Copyright 2017-2021 Intel Corporation.
+ * (C) Copyright 2017-2024 Intel Corporation.
```

Comment:
add invocations to utils/utest.yaml ?

Reply:
Indeed, it always better to have such application test integrated in non regression tests.
Is it OK for you, if I add these non regression tests in the follow-up PR to not make this PR too big ?